octavia_f5/tests/unit/controller/worker/test_l2_sync_manager.py

```diff
@@ -100,12 +100,12 @@ def test_ensure_l2_flow_all_available(self, mock_get_network,
         ]
         self.manager.ensure_l2_flow(mocked_selfips, 'test-network-id')
         mock_l2_flow.assert_called_once_with(data={
-            'test-guest-hostname_0': {
+            self.manager._bigips[0]: {
```

Comment:
How is this change related to this PR? I would ask you to keep it away (especially if it is not changing anything).

The same for all other lines in this file.

Reply:
This change is necessary. The keys of the `data` dictionary don't contain the BigIP hostnames anymore, but the `BigIPRestClient` objects themselves. So these tests fail if the names aren't replaced with references to those objects.

Reply:
Ok, got it. But anyway, I pushed a stacked PR that does not require these changes.

Reply:
Fixed in new commit, please review.